travvy/src/services/AttractionsService.js

```diff
@@ -3,8 +3,8 @@ import axios from 'axios'
 
 export default {
   // It makes get request to recommend attractions based on city
-    recommend(city,groupSize,startDate,endDate,user) {
-        return axios.get('/recommend', {params:city,groupSize,startDate,endDate,user})  
+    recommend(city,groupSize,startDate,endDate,user,preference) {
+        return axios.get('/recommend', {params:city,groupSize,startDate,endDate,user,preference})  
```

Comment:
What is "preference" referring to here?

Reply:
It refers to the value that is "Yes" if the user would like their preferences ignored. If its "Yes" the back end logic is triggered to recommend attractions without the preferences being taken into account.

Reply:
that makes sense! thanks for explaining